extension/manifest.json

```diff
@@ -402,7 +402,7 @@
 			"run_at": "document_end"
 		},
 		{
-			"matches": ["https://www.torn.com/page.php?sid=bookie*"],
+			"matches": ["https://www.torn.com/bookies.php*"],

```

Comment:
Both urls should be used probably. `bookies.php` is bookies 2.0, while `page.php?sid=bookie` is bookies 2.1 beta which will replace the other url at some time in the future.

This bookies 2.1 (open) beta was announced in the casino subforum, so it's not confidential in any way either.